.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -2,7 +2,7 @@
 name: Feature request
 about: Suggest an idea for this project.
 title: "[Feature Request] "
-labels: enhancement, Feature Request
```

Comment:
Did you mean to fully remove "feature request", or did you mean to just uncapitalize?

Reply:
remove it. identitymodel doesn't currently have that label.